xcell/mappers/mapper_Planck_base.py

```diff
@@ -29,10 +29,10 @@ def _get_Planck_defaults(self, config):
     def get_signal_map(self):
         if self.signal_map is None:
             signal_map = hp.read_map(self.file_map)
+            signal_map[signal_map == hp.UNSEEN] = 0.0
+            signal_map[np.isnan(signal_map)] = 0.0
```

Comment:
I'm thinking, shouldn't this also be done in the `_get_hm_maps` method of `MapperSPT`?

Reply:
I think it make sense, otherwise, we would be applying the masking to the base map which borrows the method from planck_base but not the HM which get their method from the child class.